src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2048,16 +2048,28 @@ export default class Aircraft {
                 const dist_to_localizer = offset[0] / sin(angle_diff); // dist from the localizer intercept point, km
 
                 if (dist_to_localizer <= turning_radius || dist_to_localizer < 0.5) {
-                    this.target.heading = angle;
-
                     // Steer to within 3m of the centerline while at least 200m out
                     if (offset[1] > 0.2 && abs(offset[0]) > 0.003) {
-                        // TODO: enumerate the magic numbers
-                        this.target.heading = _clamp(degreesToRadians(-30), -12 * offset_angle, degreesToRadians(30)) + angle;
+                        const severity_of_correction = 12;  // controls steepness of heading adjustments during localizer tracking
+                        const tgtHdg = angle + (offset_angle * -severity_of_correction);
+                        let minHdg = null;
+                        let maxHdg = null;
+                        if (!assignedHdg) {  // interception via fixes
+                            // TODO: uh.... do this.
```

Comment:
yeah it's possible to get in there. People using fixes to put a/c on the loc WOULD get there. And here it would just fail to intercept, which I guess a silent failure is a doofy solution. I intend to get some full logic in there, but I suppose it should at least have had a warning or error of some sort so when it doesn't work, it is clear why.